### PR TITLE
Add try login box

### PR DESCRIPTION
**Pull request by tung2744**
ref #3286

Screenshot 2023-09-15 at 9 32 43 PM

### CODE REVIEW

File: pkg/portal/graphql/app_mutation.go [hunk 1]
```diff
@@ -634,6 +634,81 @@ var _ = registerMutationField(
...
+var generateTestTokenInput = graphql.NewInputObject(graphql.InputObjectConfig{
+	Name: "GenerateTestTokenInput",
+	Fields: graphql.InputObjectConfigFieldMap{
+		"id": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.ID),
+			Description: "ID of the app.",
+		},
+		"returnUri": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(graphql.String),
+			Description: "URI to retrun to in the tester page",
```
Comment (louischan-oursky) [inline pkg/portal/graphql/app_mutation.go:646]: typo

File: pkg/portal/graphql/app_mutation.go [hunk 2]
```diff
@@ -634,6 +634,81 @@ var _ = registerMutationField(
...
+		Args: graphql.FieldConfigArgument{
+			"input": &graphql.ArgumentConfig{
+				Type: graphql.NewNonNull(generateTestTokenInput),
+			},
+		},
+		Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+			// Access Control: authenicated user.
+			sessionInfo := session.GetValidSessionInfo(p.Context)
+			if sessionInfo == nil {
+				return nil, AccessDenied.New("only authenticated users can visit secrets")
```
Comment (louischan-oursky) [inline pkg/portal/graphql/app_mutation.go:675]: The message is irrelevant to this mutation.

**Comment by tung2744** [comment]
@louischan-oursky Updated with the auth ui side changes, please check. Thanks!

**Review by louischan-oursky — COMMENTED** [review]
1. Text overflow the parent. See the attached image
Screenshot 2023-09-19 at 15 02 03
.
2. After code exchange, refreshing the page shows 404. I think we should show the user info of the current user.

**Comment by tung2744** [comment]
@louischan-oursky 

> Text overflow the parent.

Please re-build the auth ui. I used a new tailwind util class which haven't used before therefore need to rebuild the css file.

**Comment by tung2744** [comment]
@louischan-oursky Updated, thanks!
Now the result will persist for 20 minutes and you can refresh the page as long as the result id were kept in query.
e747b89
51d33aa
7f54d9d

**Pull request merged**